tests/unit/util/test_prometheus_exporter.py

```diff
@@ -10,9 +15,7 @@
 class TestPrometheusStatsExporter:
     def setup_method(self):
         REGISTRY.__init__()
-
-    def test_correct_setup(self):
-        status_logger_config = {
+        self.status_logger_config = {
```

Comment:
this is shared for all tests, isn't it? if yes -> why?

Reply:
The status_logger_config is used for almost all tests. I consider this now as the base configuration that is used in every test, which focus isn't the configuration. In test cases where the configuration is part of the actual test I configured a separate status_logger_config object.